Client/Frontend/Settings/Clearables.swift

```diff
@@ -46,6 +49,9 @@ class HistoryClearable: Clearable {
             CSSearchableIndex.default().deleteAllSearchableItems()
             NotificationCenter.default.post(name: .PrivateDataClearedHistory, object: nil)
             log.debug("HistoryClearable succeeded: \(success).")
+            
+            self.tabManager.clearAllTabsHistory()
```

Comment:
nit: `self.` not necessary

Reply:
Inside the closure? I think we might need it. Or, we can do a `[weak self]` and weakify self after. Which would you like?

Reply:
Argh. Missed it was in a closure 🙈